Helper/Session.php

```diff
@@ -30,7 +30,7 @@
 use Magento\Framework\Data\Form\FormKey;
 use Bolt\Boltpay\Helper\Config as ConfigHelper;
 use Bolt\Boltpay\Model\EventsForThirdPartyModules;
-use Zend\Serializer\Adapter\PhpSerialize as Serialize;
+use Magento\Framework\Serialize\Serializer\Serialize as Serialize;
```

Comment:
@danac-gs I'm not sure if this works in Magento 2.4.3, but we can't use the class \Magento\Framework\Serialize\Serializer\Serialize in the Magento older version (2.3) because the unserialized method from the Magento framework doesn't unserialize objects.
You can take a look  here \Magento\Framework\Serialize\Serializer\Serialize::unserialize for more details.
See screenshot here: https://prnt.sc/1q6wlu6

I switched to your branch, tested and I can't open the Bolt popup. It's stuck on loading 
See here: https://prnt.sc/1q6x6w8

Can you test the PR on Magento 2.3?

Reply:
Thanks @johnmccombs1 , will find another solution.